modules/default/calendar/calendarfetcher.js

```diff
@@ -41,387 +44,383 @@ const CalendarFetcher = function (url, reloadInterval, excludedEvents, maximumEn
 	 * Initiates calendar fetch.
 	 */
 	const fetchCalendar = function () {
-		clearTimeout(reloadTimer);
-		reloadTimer = null;
-
-		const nodeVersion = Number(process.version.match(/^v(\d+\.\d+)/)[1]);
-		const opts = {
-			headers: {
+		function getFetcher(url, auth) {
```

Comment:
why make a function here that is only used once?

Reply:
willt take a lokk at this later

Reply:
function is removed